docs/mv-guide.md

```diff
@@ -0,0 +1,342 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to you under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+# CarbonData Materialized View
+
+* [Quick Example](#quick-example)
+* [Introduction](#introduction)
+* [Loading Data](#loading-data)
+* [Querying Data](#querying-data)
+* [Compaction](#compacting)
+* [Data Management](#data-management)
+* [Time Series Support](#time-series-support)
+* [Time Series RollUp Support](#time-series-rollup-support)
+
+## Quick example
+
+ Start spark-sql in terminal and run the following queries,
+
+   ```
+     CREATE TABLE maintable(a int, b string, c int) stored as carbondata;
+     INSERT INTO maintable SELECT 1, 'ab', 2;
+     CREATE MATERIALIZED VIEW view1 AS SELECT a, sum(b) FROM maintable GROUP BY a;
+     SELECT a, sum(b) FROM maintable GROUP BY a;
+     // NOTE: run explain query and check if query hits the Index table from the plan
+     EXPLAIN SELECT a, sum(b) FROM maintable GROUP BY a;
+   ```
+
+## Introduction
+
+ Materialized views are created as sub-queries. User can create limitless materialized view to 
```

Comment:
created as sub-queries? Please change the statement

Reply:
mv is carbon table, but not an internal table, user can query it directly.

Reply:
then, would be `Materialized views are created as carbon tables from select queries.`

Reply:
@niuge01 Please fix the above comment

Reply:
OK